osu.Game/Skinning/LegacySkin.cs

```diff
@@ -52,22 +53,29 @@ protected override void Dispose(bool isDisposing)
 
         public override Drawable GetDrawableComponent(string componentName)
         {
+            bool animatable = false;
+            (bool looping, double frametime) animationData = (false, 1000 / 60d);
```

Comment:
why not constant for frametime? (`default_animation_frametime`)

Reply:
and why a tuple..?